data/templates/cloud-config.yaml

```diff
@@ -0,0 +1,35 @@
+#cloud-config
+write_files:
+  - path: /etc/rc.local
+    permissions: "0755"
+    owner: root
+    content: |
+      #!/bin/bash
+      modprobe ipmi_devintf
```

Comment:
trying to start ipmi driver. ipmi cataloging will fail if driver doesn't exist or fail to start.